website/pages/docs/enterprise/sentinel/properties.mdx

```diff
@@ -30,7 +30,7 @@ The following properties are available in the `request` namespace.
 
 | Name                     | Type                  | Description                                                                                 |
 | :----------------------- | :-------------------- | :------------------------------------------------------------------------------------------ |
-| `connection.remote_addr` | `string`              | TCP/IP source address/port of the client                                                    |
+| `connection.remote_addr` | `string`              | TCP/IP source IP of the client                                                    |
```

Comment:
"TCP/IP source IP" sounds a bit redundant, we should keep the address terminology here.

```suggestion
| `connection.remote_addr` | `string`              | TCP/IP source address of the client                                                    |
```